NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty
# Bb display tags

Description
When you click on Tags in the NavBar you will be able to see the list of Tags

Testing Instructions

git add and git commit whatever you're working on on your current branch.
git checkout main
pull any changes to main and get all branches.
git checkout bb-display-tags
If you do not have Tabloid MVC database please use these [SQL Seeds](https://github.com/NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty/tree/main/SQL)
After you have done these steps run the main code in VS Community (Purple)
Then run "npm start" for your front end code.
Once you get into the main page login with this email "[foo@bar.com](mailto:foo@bar.com)"
Then when you are logged in you will click on "Tags" at the top of the page on the navbar and you will be able to see the Tags listed in Alphabetical order.
Checklist:
[x] My code follows the style guidelines of this project
[x]I have performed a self-review of my own code
[ ] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
